src/sql/workbench/contrib/notebook/browser/outputs/gridOutput.component.ts

```diff
@@ -200,7 +200,7 @@ class DataResourceTable extends GridTableBase<any> {
 		@IUntitledTextEditorService untitledEditorService: IUntitledTextEditorService,
 		@IConfigurationService configurationService: IConfigurationService
 	) {
-		super(state, createResultSet(source), { actionOrientation: ActionsOrientation.HORIZONTAL }, contextMenuService, instantiationService, editorService, untitledEditorService, configurationService);
+		super(state, createResultSet(source), { actionOrientation: ActionsOrientation.VERTICAL }, contextMenuService, instantiationService, editorService, untitledEditorService, configurationService);
```

Comment:
Did we talk to design about this change?

Reply:
This is same as query experience
![image](https://user-images.githubusercontent.com/29285871/102150913-82426200-3e26-11eb-97dd-238490404d0f.png)

Reply:
reverted this